### ISSUE TITLE
MathJax field should not move, or at least not a lot

### ISSUE THREAD

**Issue by Arthur-Milchior** [ankitects/anki#2092]
Let's say I type `\int\int_S\hat{\mathbf n}\cdot\nabla\times \mathbf FdS`. Let's imagine I even make some typo and should correct what I did because I pressed a wrong key.
Currently, the mathjax editor makes very hard to follow what's going on on the screen, because it moves a lot when I type.
If I only add new letters, let's say "0123456789", it regularly moves to the right, that's smooth and I can follow, even if it's not ideal.
But let's say I now add some "\mathbf" or "{", suddenly mathjax fails to compile, the font change, become small and red on yellow background. This means that suddenly the mathjax windows moves a lot on the left, because the mathjax rendering is now less wide. Then I enter the letter that should be in blackboard font, or I enter the "}", the mathjax expression is valid, and the mathjax window jumps back to the right.
Worse, sometime, for reason I don't clearly understand, the mathjax windows jumps below the mathjax expression instead of remaining on its right.

The trouble in all of those cases is that it becomes hard to follow the cursor, and to figure out what will occur. Whether I should press back or delete, whether I'm at the right place to type the new text. If I mistyped a letter, to see what was actually typed. Because typing when the field jumps a lot is extremely perturbing

**Comment by dae** [comment]
Does the jumping still happen with latest main, now that the images are no longer subject to shrinking?
